cloudant-sync-datastore-android-encryption/src/main/java/com/cloudant/sync/sqlite/android/AndroidSQLCipherSQLite.java

```diff
@@ -47,7 +49,7 @@
      * @param provider Provider object that contains the key to encrypt the SQLCipher database
      * @return
      */
-    public static AndroidSQLCipherSQLite createAndroidSQLite(String path, KeyProvider provider) {
+    public static AndroidSQLCipherSQLite openOrCreate(File path, KeyProvider provider) {
```

Comment:
This method implies, that if the database exists, we'll open it, if it doesn't we'll just create it. I think we need a better name, not sure what though.